go_api/util.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"net/smtp"
+	"time"
+)
+
+func GenerateToken() string {
+	rand.Seed(time.Now().UnixNano())
+	const TOKEN_CHAR_SET = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
+	token := [4]byte{}
+	for i := 0; i < 4; i++ {
+		token[i] = TOKEN_CHAR_SET[rand.Intn(len(TOKEN_CHAR_SET))]
+	}
+
+	return string(token[:])
+}
+
+func SendEmail(from string, to []string, message string) {
```

Comment:
Should we have a `SendEmail` and `SendVerification`?

I think it would be cool to be able to send emails when logged in

Reply:
Yeah true, lets make a general one and one for verification.